app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import json
+from decimal import Decimal
+
+
+def calculate_profit(name: str):
+    with open(name) as f:
+        data = json.load(f)
+        bought = 0
+        sold = 0
+        bought_account = 0
+        sold_account = 0
+        for trans in data:
+            if trans["bought"] is not None:
+                bought += Decimal(trans["bought"])
+                bought_account += \
+                    Decimal(trans["bought"]) * \
+                    Decimal(trans["matecoin_price"])
+            bought += 0
```

Comment:
Why do you need this?

Reply:
I was shocked, when saw this line. Don't really know why I wrote it